[bug]: (new-design) Dependent packages not installed when "From Titan" selected

### What seems to be the problem? 🤔

When running `npx @freesewing/new-design@next` using `yarn` and selecting "From Titan", the resulting freestanding development environment is missing packages that Titan depends on. For example,
```
Module not found: Can't resolve '@freesewing/snapseries'
```

I have not tested the other "From <design>" options to see if/which packages are similarly missing. This problem is not seen with the "From scratch" option.

### Design / Plugin / Package 🧐

packages/new-design

### Are you a FreeSewing patron? 😃

No, I am not 😞

### Additional context

A workaround is to edit `package.json` to manually add the packages to the depenencies, and then running `yarn install`:
```diff
   "dependencies": {
     "@freesewing/core": "next",
-    "@freesewing/titan": "next"
+    "@freesewing/titan": "next",
+    "@freesewing/brian": "next",
+    "@freesewing/plugin-bundle": "next",
+    "@freesewing/plugin-bust": "next",
+    "@freesewing/snapseries": "next"
   },
```

(I am not sure if all of these are first-order `Dependencies` or whether any are actually `devDependencies`.)

--- a/packages/new-design/templates/from-bella/package.json.mustache
+++ b/packages/new-design/templates/from-bella/package.json.mustache
@@ -35,6 +35,7 @@
   "dependencies": {
     "@freesewing/core": "{{ tag }}",
     "@freesewing/bella": "{{ tag }}",
+    "@freesewing/plugin-bundle": "{{ tag }}"
   },
   "devDependencies": {
     "@freesewing/plugin-svgattr": "{{ tag }}",

--- a/packages/new-design/templates/from-bent/package.json.mustache
+++ b/packages/new-design/templates/from-bent/package.json.mustache
@@ -34,7 +34,10 @@
   },
   "dependencies": {
     "@freesewing/core": "{{ tag }}",
-    "@freesewing/bent": "{{ tag }}"
+    "@freesewing/bent": "{{ tag }}",
+    "@freesewing/brian": "{{ tag }}",
+    "@freesewing/plugin-bundle": "{{ tag }}",
+    "@freesewing/plugin-bust": "{{ tag }}"
   },
   "devDependencies": {
     "@freesewing/plugin-svgattr": "{{ tag }}",

--- a/packages/new-design/templates/from-breanna/package.json.mustache
+++ b/packages/new-design/templates/from-breanna/package.json.mustache
@@ -34,7 +34,8 @@
   },
   "dependencies": {
     "@freesewing/core": "{{ tag }}",
-    "@freesewing/breanna": "{{ tag }}"
+    "@freesewing/breanna": "{{ tag }}",
+    "@freesewing/plugin-bundle": "{{ tag }}"
   },
   "devDependencies": {
     "@freesewing/plugin-svgattr": "{{ tag }}",

--- a/packages/new-design/templates/from-brian/package.json.mustache
+++ b/packages/new-design/templates/from-brian/package.json.mustache
@@ -34,7 +34,9 @@
   },
   "dependencies": {
     "@freesewing/core": "{{ tag }}",
-    "@freesewing/brian": "{{ tag }}"
+    "@freesewing/brian": "{{ tag }}",
+    "@freesewing/plugin-bundle": "{{ tag }}",
+    "@freesewing/plugin-bust": "{{ tag }}"
   },
   "devDependencies": {
     "@freesewing/plugin-svgattr": "{{ tag }}",

--- a/packages/new-design/templates/from-titan/package.json.mustache
+++ b/packages/new-design/templates/from-titan/package.json.mustache
@@ -34,7 +34,9 @@
   },
   "dependencies": {
     "@freesewing/core": "{{ tag }}",
-    "@freesewing/titan": "{{ tag }}"
+    "@freesewing/titan": "{{ tag }}",
+    "@freesewing/snapseries": "{{ tag }}",
+    "@freesewing/plugin-bundle": "{{ tag }}"
   },
   "devDependencies": {
     "@freesewing/plugin-svgattr": "{{ tag }}",

